includes/discovery/processors/comware.inc.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2017 Søren Friis Rosiak <sorenrosiak@gmail.com>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+if ($device['os'] == 'comware') {
+    echo 'Comware ';
+
+    $x = 1;
+    $entphydata = dbFetchRows("SELECT `entPhysicalIndex` FROM `entPhysical` WHERE `device_id` = ? AND `entPhysicalName` LIKE '%Board%' ORDER BY `entPhysicalIndex`", array($device['device_id']));
```

Comment:
We could just poll entPhysical directly, but seems a bit silly.
Only other way is to poll h3cEntityExtMemSize(Yeah I know..) as I see it.